iree/compiler/Dialect/HAL/Target/LLVM/AOT/internal/UnixLinkerTool.cpp

```diff
@@ -31,7 +31,43 @@ class UnixLinkerTool : public LinkerTool {
 
   std::string getToolPath() const override {
     auto toolPath = LinkerTool::getToolPath();
-    return toolPath.empty() ? "ld.lld" : toolPath;
+    if (!toolPath.empty()) return toolPath;
+    if (targetTriple.isAndroid()) {
+// TODO(ataei, benvanik): Windows cross-linking android NDK support.
+#if defined(__unix__) && defined(__x86_64__)
+#if defined(__linux__)
```

Comment:
Can you use these instead?
https://github.com/google/iree/blob/551f254a6fd3d017c3e05e8eeb5056fbe77d886c/iree/base/target_platform.h#L42-L50

Reply:
Cool! will update the PR to use these.